client/src/components/JobCard/index.tsx

```diff
@@ -0,0 +1,85 @@
+import './style.css'
+import { useEffect, useState } from "react";
+import { Link } from 'react-router-dom';
+import { Grid, Box, Card, CardMedia, CardContent, Typography } from '@material-ui/core';
```

Comment:
MUI component imports should be `from '@mui/material`
check slack for my message on this